Source/Scene/Cesium3DTilePointFeature.js

```diff
@@ -0,0 +1,789 @@
+            },
+            set : function(value) {
+                this._label.backgroundPadding = value;
+            }
+        },
+
+        /**
+         * Gets or sets whether to display the background of the text for this feature.
+         * <p>
+         * Only applied when <code>labelText</code> is defined.
+         * </p>
+         *
+         * @memberof Cesium3DTilePointFeature.prototype
+         *
+         * @type {Boolean}
+         */
+        backgroundEnabled : {
+            get : function() {
+                return this._label.showBackground;
+            },
+            set : function(value) {
+                this._label.showBackground = value;
+            }
+        },
+
+        /**
+         * Gets or sets the near and far scaling properties for this feature.
+         *
+         * @memberof Cesium3DTilePointFeature.prototype
+         *
+         * @type {NearFarScalar}
+         */
+        scaleByDistance : {
+            get : function() {
+                return this._label.scaleByDistance;
+            },
+            set : function(value) {
+                this._label.scaleByDistance = value;
+                this._billboard.scaleByDistance = value;
+            }
+        },
+
+        /**
+         * Gets or sets the near and far translucency properties for this feature.
+         *
+         * @memberof Cesium3DTilePointFeature.prototype
+         *
+         * @type {NearFarScalar}
+         */
+        translucencyByDistance : {
+            get : function() {
+                return this._label.translucencyByDistance;
+            },
+            set : function(value) {
+                this._label.translucencyByDistance = value;
+                this._billboard.translucencyByDistance = value;
+            }
+        },
+
+        /**
+         * Gets or sets the condition specifying at what distance from the camera that this feature will be displayed.
+         *
+         * @memberof Cesium3DTilePointFeature.prototype
+         *
+         * @type {DistanceDisplayCondition}
+         */
+        distanceDisplayCondition : {
+            get : function() {
+                return this._label.distanceDisplayCondition;
+            },
+            set : function(value) {
+                this._label.distanceDisplayCondition = value;
+                this._polyline.distanceDisplayCondition = value;
+                this._billboard.distanceDisplayCondition = value;
+            }
+        },
+
+        /**
+         * Gets or sets the height offset in meters of this feature.
+         *
+         * @memberof Cesium3DTilePointFeature.prototype
+         *
+         * @type {Number}
+         */
+        heightOffset : {
+            get : function() {
+                return this._heightOffset;
+            },
+            set : function(value) {
+                var offset = defaultValue(this._heightOffset, 0.0);
+
+                // TODO: ellipsoid
```

Comment:
Yes, we somehow need a reference to the ellipsoid. Since features are created the first time one is requested, is there clean way to get a reference to the ellipsoid?